app/src/main/java/org/simple/clinic/drugs/PrescribedDrug.kt

```diff
@@ -60,7 +58,7 @@ data class PrescribedDrug(
         createdAt = createdAt,
         updatedAt = updatedAt,
         deletedAt = deletedAt,
-        recordedAt = recordedAt)
+        recordedAt = createdAt)
```

Comment:
Shouldn't we be removing this property from the payloads as well?

Reply:
This will be done in another PR.